pkgs/development/python-modules/mimesis/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pytest, pytest-flake8, pytest-isort, pytest-mock, pytz }:
+
+buildPythonPackage rec {
+  pname = "mimesis";
+  version = "3.3.0";
```

Comment:
```suggestion
  version = "3.3.0";
  disabled = isPy27;
```

Reply:
good point, missed this package is Python 3. let me update